docs/creating-studies.md

```diff
@@ -19,7 +19,7 @@ you'll be working on study development. `cumulus-library` will look in each
 subdirectory of that folder for manifest files, so you can run several studies
 at once. 
 
-If you're not doing this, you can always add the `--study_dir path/to/dir` argument
+If you're not doing this, you can always add the `--study-dir path/to/dir` argument
 to any build/export call to tell it where to look for your work.
 
 ## Creating a new study
```

Comment:
a lot of this section is rewritten on my branch - we'll hammer it out over there.